src/components/login/view.rs

```diff
@@ -305,9 +305,12 @@ fn Page2<'a>(
                     style: "text-decoration: underline; cursor: pointer;",
                     onclick: move |_| {
                         if let Some(url) = view_store.app_data.as_ref().and_then(|e| e.url.clone()) {
-                            if let Ok(mut ctx) = copypasta::ClipboardContext::new() {
-                                let _ = ctx.set_contents(url);
-                            }
+                             _ = match copypasta::ClipboardContext::new() {
```

Comment:
Need to figure out how to set the `view_store.error_message`.

Reply:
Hey! You'd define a new item on the `Action` enum for the Login and then send it with the error message, like so:

https://github.com/terhechte/Ebou/blob/main/src/components/conversation/view.rs#L80

If the Error Message can't be cloned and can't be send you could format it into a string first